internal/oci/runtime_oci.go

```diff
@@ -638,19 +642,24 @@ func (r *runtimeOCI) StopContainer(ctx context.Context, c *Container, timeout in
 		}
 	}
 
-	return waitContainerStop(ctx, c, killContainerTimeout, false)
+	err = waitContainerStop(ctx, c, killContainerTimeout, false)
```

Comment:
This block should be reverted to previous version.

Reply:
Addressed your 3 comments.